src/t8_element_cxx.hxx

```diff
@@ -768,6 +768,18 @@ struct t8_eclass_scheme
   virtual void
   t8_element_destroy (int length, t8_element_t **elem) const
     = 0;
+
+  virtual int
+  t8_element_pack (const t8_element_t *elements, int count, void *send_buffer, int buffer_size, int *position,
```

Comment:
Since these are modelled after the original `MPI_Pack` and `MPI_Unpack` functions we should stick to the original parameter types as well.
And the originals do not use const.

Reply:
The const in the header file for a variable passed by value does nothing.
There is a difference in the c file, which indicates that the variable passed by value can't be changed. This can be useful for when you want to have better compiler error messages when you write `if (a=b)` instead of `if(a==b)`.
But it doesn't change the interface, as the variable is passed by value anyway.